### PR TITLE
content changes and rpa details macro

### PR DESCRIPTION
**Pull request by liamwilson-kainos**
Covers tickets - 

image

* introduces RPA contact details reusable macro

### CODE REVIEW

File: app/views/claim-success.njk [hunk 1]
```diff
@@ -12,17 +12,11 @@
     <p>Your claim has been successfully submitted.</p>
     <p>We've sent you a confirmation email.</p>
     <h2 class="govuk-heading-m">What happens next</h2>
-    <p>The Rural Payments Agency may contact you to request the review summary the vet has given you.</p>
+    <p>The Rural Payments Agency (RPA) may contact you to request the review summary the vet has given you.</p>
     <h2 class="govuk-heading-m">Get help with your claim</h2>
     <p>Contact the Rural Payments Agency if you need to change the details of your claim - for example if you've entered the vet's Royal College of Veterinary Surgeons number incorrectly.</p>
```
Comment (chrissmKainos) [inline app/views/claim-success.njk:17]: This should be _RPA_ and not _Rural Payment Agency_

File: app/views/select-your-business.njk [hunk 2]
```diff
@@ -12,9 +12,7 @@
     <form method="POST" autocomplete="off" novalidate="novalidate">
       <input type="hidden" name="crumb" value="{{crumb}}"/>
 
-      {{ govukInsetText({
-        text: "Select the business that’s claiming:"
-      }) }}
+      <p>Select the business that’s claiming:</p>
```
Comment (chrissmKainos) [inline app/views/select-your-business.njk:15]: Is this change covered by a ticket? I know there was one for apply.

File: app/routes/check-answers.js [hunk 3]
```diff
@@ -32,12 +32,12 @@ module.exports = {
           actions: { items: [{ href: '/claim/vet-visit-date', text: 'Change', visuallyHiddenText: 'change date of visit' }] }
         },
         {
-          key: { text: 'Vet name' },
+          key: { text: 'Vet\'s name' },
```
Comment (chrissmKainos) [inline app/routes/check-answers.js:35]: Which ticket covers this content change?

File: app/lib/display-helpers.js [hunk 4]
```diff
@@ -11,10 +11,10 @@ function getTypeOfReviewForDisplay (claimData) {
 
 function getSpeciesEligbileNumberForDisplay (claimData) {
   return {
-    beef: 'More than 10 beef cattle ',
-    dairy: 'More than 10 dairy cattle',
-    pigs: 'More than 50 pigs ',
-    sheep: 'More than 20 sheep '
+    beef: '11 beef cattle or more ',
```
Comment (chrissmKainos) [inline app/lib/display-helpers.js:14]: Which ticket covers this content change?

**Review by chrissmKainos — CHANGES_REQUESTED** [review]
Added a few general comments re ticket coverage and also one that requires a change.

File: app/views/select-your-business.njk (hunk 2 above)
Comment (liamwilson-kainos) [reply app/views/select-your-business.njk:15]: Yeah did not strictly see a ticket - but saw it in the prototype -> https://fcp-ahwr-prototype.herokuapp.com/v6/farmer/claim/choose-sbi

Will ask Zohaib to make a ticket retrospectively to cover testing effort

File: app/routes/check-answers.js (hunk 3 above)
Comment (liamwilson-kainos) [reply app/routes/check-answers.js:35]: Highlighted in yellow in all "species" tickets e.g. https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/169090

Hmmm but not in the prototype :(

File: app/lib/display-helpers.js (hunk 4 above)
Comment (liamwilson-kainos) [reply app/lib/display-helpers.js:14]: https://fcp-ahwr-prototype.herokuapp.com/v10/defraid-claim/defraid-1/check-answers

https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/169090

One ticket per species

File: app/views/claim-success.njk (hunk 1 above)
Comment (liamwilson-kainos) [reply app/views/claim-success.njk:17]: hmmm later version prototype does not show that -> https://fcp-ahwr-prototype.herokuapp.com/v10/defraid-claim/defraid-1/confirmation

**Review by marcin-mogiela — APPROVED** [review]

File: app/views/claim-success.njk (hunk 1 above)
Comment (liamwilson-kainos) [reply app/views/claim-success.njk:17]: Corrected!

**Review by govTechSatish — APPROVED** [review]
LGTM

**Review by govTechSatish — APPROVED** [review]
LGTM

**Pull request merged**